src/orders/views/OrderManualTransationRefund/OrderManualTransationRefund.tsx

```diff
@@ -0,0 +1,81 @@
+import {
+  OrderTransactionRequestActionMutation,
+  TransactionActionEnum,
+  useOrderTransactionRequestActionMutation,
+  useOrderTransationsDataQuery,
+} from "@dashboard/graphql";
+import useNavigator from "@dashboard/hooks/useNavigator";
+import useNotifier from "@dashboard/hooks/useNotifier";
+import { OrderManualTransationRefundPage } from "@dashboard/orders/components/OrderManualTransationRefundPage";
+import { orderUrl } from "@dashboard/orders/urls";
+import {
+  getOrderTransactionErrorMessage,
+  transactionRequestMessages,
+} from "@dashboard/utils/errors/transaction";
+import React from "react";
+import { useIntl } from "react-intl";
+
+import { filterRefundTransactions } from "./filter";
+
+interface OrderManualTransationRefundProps {
+  orderId: string;
+}
+
+const OrderManualTransationRefund = ({
+  orderId,
+}: OrderManualTransationRefundProps) => {
+  const notify = useNotifier();
+  const navigate = useNavigator();
+  const intl = useIntl();
+
+  const { data, loading } = useOrderTransationsDataQuery({
+    displayLoader: true,
+    variables: {
+      orderId,
+    },
+  });
```

Comment:
why do you need to load transactions here? does the entire view depend on it? seems like it is only one component down the component tree, so it can be perhaps colocated?